.github/workflows/deep-fuzz-and-invariant.yml

```diff
@@ -0,0 +1,31 @@
+name: Deep Fuzz and Invariant Tests
+on:
+  push:
+    branches:
+      - development
+    paths:
+      - '**.sol'
+
+jobs:
+  deep-fuzz-and-invariant:
+    name: Run Deep Fuzz and Invariant Tests
+    runs-on: ubuntu-22.04
+    env:
+      FOUNDRY_PROFILE: intense
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
```

Comment:
Hi, I am slowly trying to raise like a phoenix from the ashes. It might be worth to upgrade all actions to v4 , here `actions/checkout@v4` , where the default runtime was upgraded to node 20.

Reply:
Ok, will upgrade it

I'm considering combining both the invariant and fuzzy configurations into a single file, as has been done here

Since setting up the invariant CI is essentially identical to the fuzzy CI setup, as it even uses the same parameters from the TOML file.
Furthermore, when running the intensive fuzzy tests, the intensive invariant tests will also run alongside them, as both are triggered by the `forge test` command, so adding a separate invariant CI configuration would result in running the same tests twice

Therefore, my idea is to delete [this](https://github.com/ubiquity/ubiquity-dollar/pull/953/files#diff-3f998c761be6559a81028a3c63428b2f900ac4c443d78740f6a0d4e91c3c1def) and [this](https://github.com/ubiquity/ubiquity-dollar/blob/0a230f977ad91bbfb01dd28bc1d2ea77faed5237/.github/workflows/deep-fuzz.yml) config file and keep a single [CI configuration](https://github.com/ubiquity/ubiquity-dollar/pull/953/files#diff-d2699a9b92f15a05174fc3183b80e6fbe98245e294b519d5e35a18645bc52589) for both

Reply:
> raise like a phoenix from the ashes

Welcome back phoenix

Reply:
Invariant and fuzz CI workflows are kept in separate files because of github CI limit of running 6 hours per a single workflow run. If we keep invariant and fuzz tests in a single workflow file we're basically limiting ourselves to 3 hours of invariant and 3 hours of fuzzing execution time while with separate workflows we have 12 available execution hours (6 hours invariant + 6 hours fuzzing).

Reply:
got it, will refactor